documentation/man/features/kw-bd.rst

```diff
@@ -0,0 +1,42 @@
+=======================================
+kw-bd - Build the kernel and deploy it
+=======================================
+
+.. _bd-doc:
+
+SYNOPSIS
+========
+
+*kw* (*bd*)
+
+DESCRIPTION
+===========
+
+If you are in a kernel directory, this command will first try to
+execute kw build, which builds the kernel, and then execute kw deploy,
+which deploys the kernel.
```

Comment:
Maybe something more direct like:

Kw bd is expected to run inside a kernel tree. It executes the build, and if there are no errors, it proceeds with deploying the new kernel.